src/commodore-docker/readme.md

```diff
@@ -0,0 +1,7 @@
+# Commodore Docker manager
+
+Extracts all container image dependencies of a Commodore component.
+
+By default, the manager only searches in `class/defaults.yml` for entries in `parameters.components.*.images`.
+The manager is guaranteed to find image dependencies that adhere to the [best practices for container image dependencies](https://syn.tools/syn/explanations/commodore-components/container-images.html).
+It will attempt to parse other common dependency structures, but might not be able to parse them correctly.
```

Comment:
Do we want to list other formats that the manager attempts to parse here, or did you intentionally leave them out?

Reply:
I was unsure if I should list them. In the end I decided not to. I see it as: We guarantee that we can renovate your images if you adhere to our best practices. Otherwise we just try on a best effort basis. This would allow us to change implementation details / remove some things we can currently find, without it being a breaking change.

But you're right, having a list might be helpful anyway.  Maybe we could add a list, but explicitly say that anything other than the best practice might break without prior notice?

Reply:
I'd honestly just link the tests 😅

Reply:
I added an overview of alternative formats, with a big notice that we only officially support the best practices